cargo.nut

```diff
@@ -254,6 +255,8 @@ function GetEconomyCargoList(economy, cargo_list) {
                 "WOOD","MAIZ","WDPR","WORK","STUD","OTI2","TRSH","VEHI","PRIS","FOOD",
                 "PASS","PETR","RUBR","PLAS","TYRE","HVEH"];
 
+    case(Economies.MINIMALIST): // Minimalist Industries
+        return ["PASS","MAIL","GOOD"]
```

Comment:
The cargos ids are not directly one after another, it needs some `null` cargo ids in between:

```suggestion
        return ["PASS", null,"MAIL", null, null,"GOOD"]
```